types/mui-datatables/index.d.ts

```diff
@@ -174,40 +200,143 @@ export interface MUIDataTableColumnState extends MUIDataTableColumnOptions {
 }
 
 export interface MUIDataTableColumnOptions {
+    /**
+     * Function that returns a string or React component.
+     * Used to display data within all table cells of a given column.
+     * The value returned from this function will be used for filtering in the filter dialog.
+     * If this isn't need, you may want to consider customBodyRenderLite instead.
+     * @link https://github.com/gregnb/mui-datatables/blob/master/examples/component/index.js
+     */
     customBodyRender?: (
-        value: string,
+        value: any,
         tableMeta: MUIDataTableMeta,
         updateValue: (value: string) => void,
     ) => string | React.ReactNode;
     /**
      * Similar to and performing better than `customBodyRender`, however with the following caveats:
      * 1. The value returned from this function is not used for filtering, so the filter dialog will use the raw data from the data array.
      * 2. This method only gives you the dataIndex and rowIndex, leaving you to lookup the column value.
+     * @link https://github.com/gregnb/mui-datatables/blob/master/examples/large-data-set/index.js
      */
     customBodyRenderLite?: (dataIndex: number, rowIndex: number) => string | React.ReactNode;
+    /**
+     * Function that returns a string or React component.
+     * Used for creating a custom header to a column.
+     * This method only affects the display in the table's header, other areas of the table (such as the View Columns popover), will use the column's label.
+     */
     customHeadLabelRender?: (dataIndex: number, rowIndex: number) => string | React.ReactNode;
+    /**
+     * These options only affect the filter chips that display after filter are selected.
+     * To modify the filters themselves, see filterOptions.
+     */
     customFilterListOptions?: MUIDataTableCustomFilterListOptions;
+    /** @deprecated use customFilterListOptions.render */
     customFilterListRender?: (value: any) => string;
+    /** Function that returns a string or React component. Used as display for column header. */
     customHeadRender?: (
         columnMeta: MUIDataTableCustomHeadRenderer,
-        updateDirection: (params: any) => any,
+        handleToggleColumn: (columnIndex: number) => void,
+        sortOrder: MUISortOptions,
     ) => string | React.ReactNode;
+    /**
+     * Determines if the column can be dragged.
+     * The draggableColumns.enabled option must also be true.
+     * @defaultValue true
```

Comment:
```suggestion
     * @default true
```

defaultValue is an alias to `default` that is very uncommon. Lets stick to what is common and most used.